airbyte-server/src/main/java/io/airbyte/server/handlers/SchedulerHandler.java

```diff
@@ -114,7 +119,8 @@ public SchedulerHandler(final ConfigRepository configRepository,
         jobPersistence,
         eventRunner,
         new JobConverter(workerEnvironment, logConfigs),
-        connectionsHandler);
+        connectionsHandler,
+        new EnvVariableFeatureFlags());
```

Comment:
Can we inject that instead of creating a new instance here, it will help for the migration to micronaut.